src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -158,7 +159,7 @@ internal partial class InProcLanguageServer : IAsyncDisposable
             var service = serviceContainer.GetFullAccessServiceBroker();
 
             var configuration = await TraceConfiguration.CreateTraceConfigurationInstanceAsync(service, cancellationToken).ConfigureAwait(false);
-            var traceSource = await configuration.RegisterLogSourceAsync(logId, new LogHub.LoggerOptions(), cancellationToken).ConfigureAwait(false);
+            var traceSource = await configuration.RegisterLogSourceAsync(logId, new RpcContracts.Logging.LoggerOptions(), cancellationToken).ConfigureAwait(false);
```

Comment:
There are a few new parameters for `LoggerOptions` that would allow you to avoid force overriding the switch level in line 164. I would suggest doing this now:

```
var logOptions = new LoggerOptions(requestedLoggingLevel: new LoggingLevelSettings(SourceLevels.ActivityTracing | SourceLevels.Information));
var traceSource = await configuration.RegisterLogSourceAsync(logId, logOptions, cancellationToken).ConfigureAwait(false);
```